tests/unit/test_profiling.py

```diff
@@ -0,0 +1,65 @@
+from multiprocessing import Process
```

Comment:
Maybe we could place this within the `tests/performance` folder, as `tests/performance/tests/test_profiling.py`, since this is not part of the `rdt` package (notice that the `unit` folder follows the same hierarchy as `rdt`, so `tests/unit/test_profiling.py` should be dedicated to testing `rdt/profiling.py`